fb_apt/attributes/default.rb

```diff
@@ -0,0 +1,22 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+if node.debian?
+  mirror = 'http://http.debian.net/debian'
+elsif node.ubuntu?
+  mirror = 'http://archive.ubuntu.com/ubuntu'
+end
```

Comment:
wanna throw mint in here while you're at it? It's forked form ubuntu and uses apt and should just work..